eks/terraform.tfvars

```diff
@@ -35,25 +28,25 @@
 # gpu_node_pool_delete_on_termination   = true
 # gpu_node_pool_root_disk_size_gb       = 512
 # gpu_node_pool_root_volume_type        = "gp2"
-# gpu_operator_driver_version           = "535.104.05"
+# gpu_operator_driver_version           = "535.129.03"
 # gpu_operator_namespace                = "gpu-operator"
-# gpu_operator_version                  = "v23.6.1"
+# gpu_operator_version                  = "v23.9.1"
 # max_cpu_nodes                         = "2"
 # max_gpu_nodes                         = "5"
 # min_cpu_nodes                         = "0"
 # min_gpu_nodes                         = "2"
 # nvaie                                 = false
-# nvaie_gpu_operator_driver_version     = "525.125.06"
-# nvaie_gpu_operator_version            = "v23.3.2"
+# nvaie_gpu_operator_driver_version     = "535.129.03"
+# nvaie_gpu_operator_version            = "v23.9.0"
 # private_subnets = [
-#   "10.0.1.0/24",
-#   "10.0.2.0/24",
-#   "10.0.3.0/24"
+#   "10.0.0.0/19",
```

Comment:
Should probably add a comment as to why these new variables are appropriate

Reply:
Someone else who made a commit to change the values didn't update the default values in the tfvars file. The actual code change was https://github.com/NVIDIA/nvidia-terraform-modules/pull/17.